build.sbt

```diff
@@ -33,7 +33,7 @@ libraryDependencies ++= Seq(
   "org.asciidoctor" % "asciidoctorj-diagram" % "1.5.4.1"
 )
 
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.3.2")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.4.3")
```

Comment:
`0.4.4` is out now.

Reply:
Okay, will try and see if that fixes the build error.